cmd/root/sweet_test.go

```diff
@@ -474,3 +475,33 @@ func TestFromArgsWithEmptyExerciseFiles(t *testing.T) {
 	}
 
 }
+
+func TestParseEvent(t *testing.T) {
+
+	type testCase struct {
+		name  string
+		input string
+		want  event
+	}
+
+	ts, _ := time.Parse("2006-01-02 15:04:05.000", "2024-10-07 13:46:47.679")
+	testCases := []testCase{
+		{
+			name:  "Easy case",
+			input: "2024-10-07 13:46:47.679: 0 a h",
+			want: event{
```

Comment:
Change this want to be a string, which I can use to parse the event in the function below. That way, I won't have to declare `ts` outside.